eXpansion/Framework/PlayersBundle/Model/Base/Gamecurrency.php

```diff
@@ -0,0 +1,1601 @@
+<?php
```

Comment:
Why is this in PlayerBundle ?

Reply:
Ooh... i had wrong config at first... i forgot to remove this.